code/game/turfs/space/space.dm

```diff
@@ -21,6 +21,9 @@
 	if(config)
 		update_starlight() //MC will initialize all the space turfs that get created before config
 
+/turf/space/Destroy()
+	return QDEL_HINT_LETMELIVE

```

Comment:
You know... I wonder what happens if a space turf gets deleted. Does it just get replaced by a new space turf?

I'm *pretty sure* it should be safe to not let them be qdel'd, but I'd test building stuff in space just to be safe.